src/components/pipelines-metrics/PipelinesAverageDuration.tsx

```diff
@@ -115,7 +165,11 @@ const PipelinesAverageDuration: React.FC<PipelinesAverageDurationProps> = ({
                 style={xAxisStyle}
                 tickFormat={xTickFormat}
               />
-              <ChartAxis dependentAxis style={yAxisStyle} />
+              <ChartAxis
+                dependentAxis
+                style={yAxisStyle}
+                tickFormat={(v) => `${v}m`}
```

Comment:
Can we return in readable format  - it shows decimal values 1.76m instead of 1m 43 s
![Screenshot from 2023-12-13 20-33-37](https://github.com/openshift-pipelines/console-plugin/assets/17771352/fe2988cf-48d4-4e0d-bb0e-2855ad8699ab)